build/php/data_py.php

```diff
@@ -29,6 +29,9 @@
 $predictString = get_json_file("predictTable.json");
 $predictTable = json_decode($predictString, true);
 
+$gpTipString = get_json_file("max-priority-fee-per-gas-estimate.json");
+$gpTip = json_decode($gpTipString, true);
```

Comment:
I've verified on local env, if the above JSON file doesn't exit, it doesn't crash the application